lib/util/tooltip.coffee

```diff
@@ -94,10 +94,16 @@ class Tooltip
       hideTimer = setTimeout (=> @hide()), @hideDelay
 
   positionOverlay: ->
-    bounding = @parent.getBoundingClientRect()
-    @view.style.bottom   = document.documentElement.clientHeight - bounding.top + 'px'
-    switch @position
-      when 'left'
-        @view.style.left = bounding.left + 'px'
-      when 'right'
-        @view.style.left = bounding.left + bounding.width -  @view.offsetWidth + 'px'
+    p = @parent
+    pos = @position
+    v = @view
+    positioner = ->
+      bounding = p.getBoundingClientRect()
+      v.style.bottom   = document.documentElement.clientHeight - bounding.top + 'px'
+      switch pos
+        when 'left'
+          v.style.left = bounding.left + 'px'
+        when 'right'
+          v.style.left = bounding.left + bounding.width -  v.offsetWidth + 'px'
+      window.requestAnimationFrame positioner
+    window.requestAnimationFrame positioner
```

Comment:
Okay, this is questionable: Frametimings are worse with having this continuous animation running as opposed to only updating the position when showing the tooltip. The animation *does* look better though, in paticular when scrolling the editor while the tooltip is visible. Not sure if that is worth it though...